doc-site/data/final-form/Form.mdx

```diff
@@ -9,40 +9,46 @@ slug: /api/@prestojs/final-form/Form.html
    handleSubmit. Most the time this is the same thing so you optionally just pass through renderable
    children and the form will be created implicitly for you.
 
+The form component rendered can be specified in [UiProvider](/api/@prestojs/ui/UiProvider.html). If
+not specified this will default to `form`.
+
 
 import { Form } from '@prestojs/final-form';
 import { User } from 'docSite/models';
 import { PropTypes } from 'docSite/components/View';
+import { Button } from 'antd';
 
 <View
     componentName="Form"
     componentProps={{
-        layout: {
-            type: PropTypes.String,
-            value: 'horizontal',
-        },
         onSubmit: {
             type: PropTypes.Function,
-            value: 'data=> console.log(data)',
+            value: 'data => console.log(data)',
         },
         children: {
-            value: `
-<Form.Item field={User.fields.firstName} />
+            value: `<Form.Item field={User.fields.firstName} />
 <Form.Item field={User.fields.lastName} />
 <Form.Item field={User.fields.email} />
+<Form.Item label=" " colon={false}>
```

Comment:
Should label be blank here? If it's not required then why include it at all?

Reply:
Without the label the button appears to the very left. With an empty label it shows in the right column in line with all the fields

Reply:
Smells like an issue with "" being falsy. I assume that's in the underlying library and not something in our control

Reply:
Yeah it's in antd